var/spack/repos/builtin/packages/py-notebook/package.py

```diff
@@ -37,42 +38,42 @@ class PyNotebook(PythonPackage):
     depends_on("py-traitlets", type=("build", "run"))
-    depends_on("py-traitlets@4.2.1:", type=("build", "run"), when="@5:")
+    depends_on("py-jupyter-core@4.6.1:", when="@6.0.3:", type=("build", "run"))
+    depends_on("py-jupyter-core@4.6.0:", when="@6.0.2", type=("build", "run"))
+    depends_on("py-jupyter-core@4.4.0:", when="@5.7.0:6.0.1", type=("build", "run"))
     depends_on("py-jupyter-core", type=("build", "run"))
-    depends_on("py-jupyter-core@4.4.0:", type=("build", "run"), when="@5.7.0:6.0.1")
-    depends_on("py-jupyter-core@4.6.0:", type=("build", "run"), when="@6.0.2")
-    depends_on("py-jupyter-core@4.6.1:", type=("build", "run"), when="@6.0.3:")
+    depends_on("py-jupyter-client@5.3.4:", when="@6.0.2:", type=("build", "run"))
+    depends_on("py-jupyter-client@5.3.1:", when="@6.0.0:6.0.1", type=("build", "run"))
+    depends_on("py-jupyter-client@5.2.0:", when="@5.7.0:5", type=("build", "run"))
     depends_on("py-jupyter-client", type=("build", "run"))
-    depends_on("py-jupyter-client@5.2.0:", type=("build", "run"), when="@5.7.0:5")
-    depends_on("py-jupyter-client@5.3.1:", type=("build", "run"), when="@6.0.0:6.0.1")
-    depends_on("py-jupyter-client@5.3.4:", type=("build", "run"), when="@6.0.2:")
     depends_on("py-ipython-genutils", type=("build", "run"))
     depends_on("py-nbformat", type=("build", "run"))
     # https://github.com/jupyter/notebook/pull/6286
-    depends_on("py-nbconvert@5:", type=("build", "run"), when="@5.5:")
+    depends_on("py-nbconvert@5:", when="@5.5:", type=("build", "run"))
     depends_on("py-nbconvert", type=("build", "run"))
-    depends_on("py-nest-asyncio@1.5:", type=("build", "run"), when="@6.4.10:")
+    depends_on("py-nest-asyncio@1.5:", when="@6.4.10:", type=("build", "run"))
     depends_on("py-ipykernel", type=("build", "run"))
-    depends_on("py-send2trash", type=("build", "run"), when="@6:")
-    depends_on("py-send2trash@1.5:", type=("build", "run"), when="@6.2.0:")
-    depends_on("py-send2trash@1.8:", type=("build", "run"), when="@6.4.10:")
-    depends_on("py-terminado@0.3.3:", type=("build", "run"), when="@:5.7.0")
-    depends_on("py-terminado@0.8.1:", type=("build", "run"), when="@5.7.0:")
-    depends_on("py-terminado@0.8.3:", type=("build", "run"), when="@6.1:")
-    depends_on("py-prometheus-client", type=("build", "run"), when="@5.7.0:")
+    depends_on("py-send2trash@1.8:", when="@6.4.10:", type=("build", "run"))
+    depends_on("py-send2trash@1.5:", when="@6.2.0:", type=("build", "run"))
+    depends_on("py-send2trash", when="@6:", type=("build", "run"))
+    depends_on("py-terminado@0.8.3:", when="@6.1:", type=("build", "run"))
+    depends_on("py-terminado@0.8.1:", when="@5.7.0:", type=("build", "run"))
+    depends_on("py-terminado@0.3.3:", when="@:5.7.0", type=("build", "run"))
+    depends_on("py-prometheus-client", when="@5.7.0:", type=("build", "run"))
+    depends_on("py-nbclassic@0.4:", when="@6.5:", type=("build", "run"))
```

Comment:
0.4.7+ now

Reply:
Ah, thanks. I took the one from `pyproject.toml` and did not crosscheck with the version in `setup.py`. This is fixed now.